### ISSUE TITLE
[Adaptive Card iOS - Input.Date.ErrorMessage.json]: While navigating through the swipe gesture user is not able to access the 'Done' button. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#189]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.3" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Input.Date.ErrorMessage.Jsons' and activate it.
7. The Card will open.
8. Perform right swipe and navigate to the 'This is a required Input.Date' date field and double tap to activate it.
9. Now, perform left swipe and navigate to the 'Month, Date and Year' spin button and using up and down swipe gesture select the desired date.
10. Now perform left or right swipe and observe the issue.

**Actual Result:**
While navigating using the swipe gesture user is not able to select the date under all date field. After selecting the date on performing the left or right swipe Done button is not accessible.

**Observation:**
This issue is working fine with keyboard.

**Expected Result:**
While navigating using swipe gesture, User should be able to select the date and on performing right swipe voice over focus should be land on the Done button.

**Note:**
This issue is repro throughout the application.

**User Impact:**
It will impact the screen reader user as they will not be able to select the date while navigating with Swipe gesture, due to this they will not able to perform the desired action.

**WCAG Reference:**
https://www.w3.org/TR/WCAG21/#meaningful-sequence

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/fe940a45-7e1a-4455-a832-0b326af44fda

### Card JSON

```JSON
NA
```

**Comment by VikrantSingh01** [comment]
@wyAbhishek  / @guptaabhishe : Can one of you help with this a11y compiant bug by EOM.

**Comment by wyAbhishek** [comment]
@vagpt @VikrantSingh01 
This seems to work for me. Attaching video recording of the same. 

https://github.com/user-attachments/assets/f41363b7-3aaf-42ea-b2b1-9d2be647d1e8

**Comment by VikrantSingh01** [comment]
Closing as working as indented.
